NeoDebuggerUI/ViewModels/InvokeWindowViewModel.cs

```diff
@@ -188,35 +188,51 @@ public InvokeWindowViewModel()
 
         public void Run()
         {
-            if (SelectedTestSequence == null)
+            if(DebuggerStore.instance.manager.IsSteppingOrOnBreakpoint)
             {
-                DebuggerStore.instance.manager.ConfigureDebugParameters(DebugParams);
                 DebuggerStore.instance.manager.Run();
             }
             else
             {
-                DebuggerStore.instance.manager.RunSequence(SelectedTestSequence);
-            }
-            StackItem result = null;
-            string errorMessage = null;
-            try
-            {
-                result = DebuggerStore.instance.manager.Emulator.GetOutput();
-            }
-            catch(Exception ex)
-            {
-                errorMessage = ex.Message;
+                if (SelectedTestSequence == null)
+                {
+                    DebuggerStore.instance.manager.ConfigureDebugParameters(DebugParams);
+                    DebuggerStore.instance.manager.Run();
+                }
+                else
+                {
+                    DebuggerStore.instance.manager.RunSequence(SelectedTestSequence);
+                }
             }
 
-            if (result != null)
+            if (DebuggerStore.instance.manager.IsSteppingOrOnBreakpoint)
```

Comment:
Why there is 2 if-elses checking the same thing? This clause must be re-checked after the first if? If so, put a comment explaining.